### ISSUE TITLE
Change URL via admin panel as in V4

### ISSUE THREAD

**Issue by mrkylegp** [invoiceninja/admin-portal#323]
This is a suggestion to bring back the ability to edit your URL via the admin panel as was possible in V4.

I found this very useful instead of ssh'ing in and modifying my .env file.

![V4](https://user-images.githubusercontent.com/52687659/131962121-ace7c9f4-f716-4345-aad0-3f099b767e57.png)

**Comment by hillelcoren** [comment]
@turbo124 do you think this issue belongs to the main invoiceninja repo?

**Comment by mrkylegp** [comment]
> @turbo124 do you think this issue belongs to the main invoiceninja repo?

Yes it might be, I may have misunderstood the usage of this admin repo.
